CMakeLists.txt

```diff
@@ -4,7 +4,18 @@ cmake_minimum_required(VERSION 3.4)
 
 include(ExternalProject)
 
-project( CCTag )
+project(CCTag VERSION 1.0.0 LANGUAGES C CXX)
```

Comment:
or we can use git tags to determine the version
http://brianmilco.blogspot.fr/2012/11/cmake-automatically-use-git-tags-as.html

:-D